### PR TITLE
Wkb shape serializer python

### PR DESCRIPTION
**Pull request by Imbruced**
## Is this PR related to a proposed Issue?
SEDONA-28

## What changes were proposed in this PR?
Fix PYthon API after adding multiple serializers.

## How was this patch tested?
Python unit tests per serializer, included in github worklow matrix

### CODE REVIEW

**Comment by Imbruced** [comment]
need to fix python worklow, it should include tests for shape and wkb serializer based on file content.

**Comment by Imbruced** [comment]
And do not know why R workflow does not work.

File: python/sedona/core/serde/shape/enums.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+
+
+class GeomEnum(Enum):
```
Comment (jiayuasu) [inline python/sedona/core/serde/shape/enums.py:4]: @Imbruced Any reason why this only enumerates a few types instead of all types?

Comment (Imbruced) [reply python/sedona/core/serde/shape/enums.py:4]: I used Shape types fro ShapeType enum within package org.apache.sedona.core.formatMapper.shapefileParser.parseUtils.shp or you meant 
```java
        SHAPE(0),
        CIRCLE(1),
        GEOMETRYCOLLECTION(2),
        ENVELOPE(3);
```

**Comment by Imbruced** [comment]
Did I broke something in R package ? WDYT is better to have shape and wkb serializers in matrix or run tests twice ? (Python API holds on serializers so it is essential to run tests twice per serializer).

**Comment by Imbruced** [comment]
Did I broke something in R package ? WDYT is better to have shape and wkb serializers in matrix or run tests twice ? (Python API holds on serializers so it is essential to run tests twice per serializer).

**Comment by Imbruced** [comment]
I can squash this PR (or we can squash while merging to master), sorry for additional commits, did anyone has a pleasure to use https://github.com/nektos/act to run workflows locally ?

**Comment by Imbruced** [comment]
To be honest, this PR and major one needs documentation update, should I include them here or we update docs in @netanel246 PR ?

**Comment by Imbruced** [comment]
@jiayuasu Can you add @netanel246 to reviewers ?

**Comment by jiayuasu** [comment]
@Imbruced R doc is in the R module folder. Let me merge this PR now and go back to @netanel246 PR.

**Pull request merged**